forms/pdf_form_fill_custom_font.go

```diff
@@ -0,0 +1,123 @@
+/*
+ * Fill PDF form via JSON file data and use custom font replacement.
+*
+* Run as: go run pdf_form_fill_custom_font.go.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/annotator"
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/fjson"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+// Example of filling PDF formdata with a form using custom font.
+func main() {
+	// Enable debug-level logging.
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
```

Comment:
can leave this commented out i think (debug logging should probably not be default on)